peril/compareForceSchema.ts

```diff
@@ -4,31 +4,28 @@ import fetch from "node-fetch"
 import { warn, danger } from "danger"
 
 // If there is a breaking change between the local schema,
-// and the current Reaction one, warn.
+// and the current Force one, warn.
 export default async () => {
-  const forcePackageJSON = await (await fetch(
-    "https://raw.githubusercontent.com/artsy/force/master/package.json"
-  )).json()
-  const reactionVersion = forcePackageJSON["dependencies"]["@artsy/reaction"]
-  const reactionSchemaUrl = `https://github.com/artsy/reaction/raw/v${reactionVersion}/data/schema.graphql`
+  const forceSchemaUrl =
+    "https://github.com/artsy/force/blob/master/data/schema.graphql"
```

Comment:
```suggestion
    "https://github.com/artsy/force/raw/master/data/schema.graphql"
```

Reply:
Thanks a lot for checking that out for me. I'm testing it against Danger again....

Reply:
... and it's not going away 😬